changelog/allMembers_import.dd

```diff
@@ -0,0 +1,7 @@
+The allMembers trait behaves more correctly with imports
+
+1. `import`s are presented using their fully qualified name.
+Previously only the first identifier of the import was present in the list,
+e.g `std` for `std.stdio`.
```

Comment:
I think it would be better to give a more specific example - for example before the following code:
```D
import std;
import std.algorithm;
import std.algorithm.iteration;
import std.stdio : writeln;
pragma (msg, [__traits(allMembers, mixin(__MODULE__))]);
```
Would print during compilation just: `["object", "std"]`, while now it should print: `["object", "std", "std.algorithm", "std.algorithm.iteration"]` (if I understand correctly).

Reply:
indeed and yes that is the right ouput. there is at least one more related PR to come so I'll come back to this when Buildkite will be green.